From 0edd2c6c69da4d825b4e8f833f020796aba960d2 Mon Sep 17 00:00:00 2001
From: Naveen M K <naveen521kk@gmail.com>
Date: Sat, 3 Apr 2021 17:46:48 +0530
Subject: [PATCH] Remove -DLUASOCKET_INET_PTON from Makefile Also, remove
 definition of LUASOCKET_INET_PTON it's already defined in ws2tcpip.h so don't
 redeclare it causes problems while compiling see
 https://github.com/diegonehab/luasocket/pull/300

Signed-off-by: Naveen M K <naveen521kk@gmail.com>
---
 texk/web2c/luatexdir/luasocket/src/inet.h   | 3 ---
 texk/web2c/luatexdir/luasocket/src/makefile | 2 +-
 2 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/texk/web2c/luatexdir/luasocket/src/inet.h b/texk/web2c/luatexdir/luasocket/src/inet.h
index 928c7eb4..b286e6ec 100644
--- a/texk/web2c/luatexdir/luasocket/src/inet.h
+++ b/texk/web2c/luatexdir/luasocket/src/inet.h
@@ -22,9 +22,6 @@
 #define LUASOCKET_INET_ATON
 #endif
 
-#if defined( __MINGW32__ ) || defined( __MINGW64__ )
-#define LUASOCKET_INET_PTON
-#endif
 
 int inet_open(lua_State *L);
 
diff --git a/texk/web2c/luatexdir/luasocket/src/makefile b/texk/web2c/luatexdir/luasocket/src/makefile
index 494baab0..19008c04 100644
--- a/texk/web2c/luatexdir/luasocket/src/makefile
+++ b/texk/web2c/luatexdir/luasocket/src/makefile
@@ -213,7 +213,7 @@ SOCKET_solaris=usocket.o
 SO_mingw=dll
 O_mingw=o
 CC_mingw=gcc
-DEF_mingw= -DLUASOCKET_INET_PTON -DLUASOCKET_$(DEBUG) \
+DEF_mingw= -DLUASOCKET_$(DEBUG) \
 	-DWINVER=0x0501 -DLUASOCKET_API='__declspec(dllexport)' \
 	-DMIME_API='__declspec(dllexport)'
 CFLAGS_mingw= -I$(LUAINC) $(DEF) -Wall -O2 -fno-common \
-- 
2.31.1.windows.1

